src/Microsoft.PowerShell.GlobalTool.Shim/GlobalToolShim.cs

```diff
@@ -26,7 +26,7 @@ public static class EntryPoint
         public static int Main(string[] args)
         {
             var currentPath = new FileInfo(System.Reflection.Assembly.GetEntryAssembly().Location).Directory.FullName;
-            var isWindows = Platform.IsWindows;
+            var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
We can directly use:
```suggestion
            var isWindows = OperatingSystem.IsWindows();
```

Reply:
fixed